bloptools/gp/scripts/benchmark.py

```diff
@@ -1,21 +1,38 @@
-
```

Comment:
In this module, I'd add a docstring on top of the file to explain how to run this file and what parameters need to be defined in advance (like `RE`, `db`, etc.)